data/remnant/remnant 2 cognizance.txt

```diff
@@ -0,0 +1,2471 @@
+				period 14
+		object
+			sprite planet/uranus
+			distance 2105
+			period 3438
+			object
+				sprite planet/luna
+				distance 235
+				period 14
+		object "Wormhole Link"
+			sprite planet/wormhole-red
+			distance 2754
+			period 389
+
+
+
+mission "Remnant: Cognizance 37"
+	name "Take data to Viminal."
+	description "The rest of the Remnant need to know what is happening. Take this data to them."
+	landing
+	source "Ssil Vida"
+	destination "Viminal"
+	to offer
+		has "Remnant: Cognizance 34: done"
+	on complete
+		event "remnant: cognizance salvage expansion"
+		conversation
+			`A crowd of Remnant cluster around your ship as you land, but it is just one who gestures quickly, asking if the rest of the team who went with you are safe. You nod, and a ripple of relief goes through the crowd. "So, what happened?" asks another.`
+			choice
+				`	(Explain everything that happened.)`
+					goto cognizanceexplanation
+				`	"I think I should tell this to the prefects first."`
+			`	The Remnant around you look amused. "Do not be concerned, <first>. Everything you say will reach them. The sooner you tell us, the sooner we can take start preparing for whatever is necessary.`
+			choice
+				`	(Explain everything that happened.)`
+					goto cognizanceexplanation
+				`	"I thought the prefects were your leaders?"`
+			` "They bear our executive responsibilities, it is true. Times of critical threats require the capacity for decisions to be made in the instant. But do not confuse the authority to make high-value decisions with being in charge of everything. The less immediate a decision is, the more likely it is to be the product of significant discussion among us all. Likewise, decisions about what constitute the policy and values of our society are things decided upon by the unified whole, not any one individual." She pauses, and smiles. "Admitedly, given your involvement in smaller projects and unfolding events, our society probably does have a certain autocratic appearance."`
+			choice
+				`	"OK, I will explain everything."`
+					goto cognizanceexplanation
+				`	"How can you reach concensus so easily?"`
+			branch cognizanceuntrusted
+				has "remnant untrusted"
+			`	She looks around with an uncertain expression, then nods. "As I am sure has occured to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be ackward to watch or gesture in the midst of some situations. While we have some vocabulary we can do without needing hands or arms, it is very limited. Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation. "To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`
```

Comment:
```suggestion
			`	She looks around with an uncertain expression, then nods. "As I am sure has occurred to you, communication by gestures has some disadvantages. We still prefer it for some tasks, but it can be awkward to watch or gesture in the midst of some situations. While we have some vocabulary we can do without needing hands or arms, it is very limited. Fortunately, one of the original abilities of our companions is related to the nervous system. I believe Dawn may have told you a little about it, but was scarce on details." She smiles wryly, then another Remnant takes up the conversation. "To put it simply, we can form ad-hoc neural networks between us, of varying levels of intensity. Without the necessity of speech or gestures, we can have entire conversations in the blink of an eye."`
```